cli/tests/integration_tests.rs

```diff
@@ -1603,6 +1597,70 @@ itest!(_060_deno_doc_displays_all_overloads_in_details_view {
   output: "060_deno_doc_displays_all_overloads_in_details_view.ts.out",
 });
 
+/// Kind of reflects `itest!()`. Note that the pty's output (which also contains
+/// stdin content) is compared against the content of the `output` path.
+#[cfg(not(windows))]
+fn itest_pty(args: &str, output: &str, input: &[u8]) {
```

Comment:
Can you move this function to the bottom of this file inside the "util" module?

Reply:
I've moved to the `test_util` crate, which I assume you meant.

Reply:
Thank you, yes.